app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationPetsController < ApplicationController
+  def new
+  end
+
+  def create
+    application = Application.find(params[:application_id])
+    pet = Pet.find(params[:pet_id])
+    application.pets << pet
+
```

Comment:
Would this be a good spot to add logic to prevent Non-adoptable OR duplicate pets from being added?